Makefile

```diff
@@ -141,6 +141,8 @@ docs/assets/diagram.png: go-diagrams/diagram.dot
 start: image
 	kubectl apply -f test/manifests/argo-events-ns.yaml
 	kubectl kustomize test/manifests | sed 's@quay.io/argoproj/@$(IMAGE_NAMESPACE)/@' | sed 's/:$(BASE_VERSION)/:$(VERSION)/' | kubectl -n argo-events apply -l app.kubernetes.io/part-of=argo-events --prune=false --force -f -
+
+	sleep 10
```

Comment:
What is the reason to sleep 10?

Reply:
We added kafka to the `test/manifests` for the functional tests. The kafka specs use a statefulset which for some odd reason don't set a ready condition the same way as deployments do. Because of this the `kubectl -n argo-events wait --for=condition=Ready` did not take the kafka statefulset into account, so the 10s accounts for this.

I am going to move the kafka stuff to only spin up (and tear down) when `make test-functional` is run, so I can remove the sleep statement here.